### DISCUSSION TITLE
Manager pages/Vue app customization

### DISCUSSION THREAD

**Discussion by erfanonline** [PiranhaCMS/piranha.core#1625] [Q&A]
- With 9.0.0, is there any way to modify/replace the CMS manager pages?
For example, if we want to remove the tag selection dropdown from post edit page.

- Similarly, any way to modify/replace the Vue app for post edit page

- Any way to modify resource strings, e.g. if we want to change a button label/place holder text for any control

Any pointer is much appreciated.

**Comment by tidyui** [comment]
Using the NuGet packages, no. But you have a few options:

1. You can put a local copy of **any** cshtml page in the manager in the same location in your application and it will override the embedded page.
2. With basic JavaScript you should be able to modify existing Vue apps as long as your script is loaded after the original.
3. Resource strings are located in the localization project stored in resx files. The intention here is for translation, not changing the labels to something else.

Regards
